We need to run gcc --version to get the gcc version on Solaris, as cc --version
runs Studio cc which doesn't support --version.

A more complete/portable version of this fix was submitted and accepted upstream
for the 0.60.0 release: https://github.com/mesonbuild/meson/pull/9298

--- meson-0.59.2/run_unittests.py
+++ meson-0.59.2/run_unittests.py
@@ -8437,7 +8437,7 @@ if is_osx():
             raise unittest.SkipTest('Prelinking not supported on Darwin.')
         if 'clang' in os.environ.get('CC', 'dummy'):
             raise unittest.SkipTest('Prelinking not supported with Clang.')
-        gccver = subprocess.check_output(['cc', '--version'])
+        gccver = subprocess.check_output(['gcc', '--version'])
         if b'7.5.0' in gccver:
             raise unittest.SkipTest('GCC on Bionic is too old to be supported.')
         testdir = os.path.join(self.unit_test_dir, '87 prelinking')
